drv/gimlet-seq-server/src/main.rs

```diff
@@ -46,25 +47,42 @@ enum Trace {
     RailsOff,
     Ident(u16),
     A1Status(u8),
+    A2LeaseSleep(u16),
     A2,
     A1Power(u8, u8),
     A0Power(u8),
     NICPowerEnableLow(bool),
     RailsOn,
     UartEnabled,
-    GetState,
     SetState(PowerState, PowerState),
+    UpdateState(PowerState),
     ClockConfigWrite,
     ClockConfigSuccess,
-    Status(u8, u8, u8),
-    PGStatus(u8, u8, u8),
-    SMStatus(u8, u8),
+    Status {
+        ier: u8,
+        ifr: u8,
+        amd_status: u8,
+        amd_a0: u8,
+    },
+    PGStatus {
+        b_pg: u8,
+        c_pg: u8,
+        nic: u8,
+    },
+    SMStatus {
+        a1: u8,
+        a0: u8,
+    },
+    ResetCounts {
+        rstn: u8,
+        pwrokn: u8,
+    },
     PowerControl(u8),
     InterruptFlags(u8),
     None,
 }
 
-ringbuf!(Trace, 64, Trace::None);
+ringbuf!(Trace, 100, Trace::None);
```

Comment:
Nit: making this a power-of-two may be a little more efficient (mask versus division).

Reply:
It does -- but the deeper history has been really, really helpful for debugging sequencer issues.

Reply:
We had not had room for 128 originally, but with some of the other changes, now we do; bumping this to 128.